src/odemis/gui/cont/correlation.py

```diff
@@ -238,14 +238,22 @@ def on_char(self, evt: wx.Event) -> None:
         """handle key presses
         :param evt: (wx.Event) the event"""
 
-        if not self.correlation_enabled():
-            logging.debug("correlation not enabled, passing event to canvas")
-            active_canvas = evt.GetEventObject()
-            active_canvas.on_char(evt)            # super event passthrough
-
+        # event data
         key = evt.GetKeyCode()
         shift_mod = evt.ShiftDown()
 
+        # pass event to canvas, if not a valid correlation key
+        valid_keys = [wx.WXK_LEFT, wx.WXK_RIGHT, wx.WXK_UP, wx.WXK_DOWN]
+        if evt.GetKeyCode() not in valid_keys:
+            logging.debug(f"invalid correlation key, passing event to canvas, key was: {key}")
```

Comment:
I'd drop this logging message. Essentially it's any key which is not an arrow... so pretty much one message for every key press. Too much.